src/MoBi.Assets/AppConstants.cs

```diff
@@ -1567,6 +1567,7 @@ public static class Captions
          public static readonly string MoleculeObserver = "Molecule Observer";
          public static readonly string ContainerObserver = "Container Observer";
          public static readonly string SelectMolecules = "Select Molecules";
+         public static readonly string LoadingApplication = "Loading Application...";
```

Comment:
This is just so that the caption in the splash screen does not show the name of the view!